ophyd/signal.py

```diff
@@ -858,15 +869,27 @@ def set_default_timeout(cls, *, timeout=2.0, connection_timeout=1.0):
         """
         if EpicsSignalBase.__any_instantiated:
             raise RuntimeError(
-                "The method EpicsSignalBase.set_default_timeout may only "
+                "The method EpicsSignalBase.set_defaults may only "
                 "be called before the first instance of EpicsSignalBase is "
                 "created. This is to ensure that all instances are created "
-                "with the same default retry setting in place.")
+                "with the same default settings in place.")
+
+        cls.__default_auto_monitor = auto_monitor
         cls.__default_connection_timeout = connection_timeout
         cls.__default_timeout = timeout
+        # The write_timeout is very different than the connection and read timeouts
+        # above. It relates to how long an action takes to complete. Any
+        # default value we choose here is likely to cause problems---either
+        # by being too short and giving up too early on a lengthy action or
+        # being too long and delaying the report of a failure.
+        cls.__default_write_timeout = write_timeout
 
-    # TODO Is there a good reason to prohibit setting these three timeout
-    # properties?
+        # TODO Is there a good reason to prohibit setting these three timeout
+        # properties?
+
+    @classmethod
+    def set_default_timeout(cls, *, **kwargs):
```

Comment:
This is a syntax error. You can either do `(cls, *args, **kwargs)` or `(cls, **kwargs)`. The latter means, "Only kwargs are accepted," which seems like your intent and the correct thing to do here, since `set_default_timeout` only ever accepted keyword arguments.

```suggestion
    def set_default_timeout(cls, **kwargs):
```

Reply:
Good catch.  One fine day, I will learn this pattern.